### PR TITLE
#1 Bootstrap NESTJS API project

### PR DESCRIPTION
**Pull request by mcristi**

### CODE REVIEW

File: nest-cli.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (paraschivionut) [inline nest-cli.json:1]: small typo in the above file's folder name `documenation => documentation`

File: package.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "typeorm": "ts-node ./node_modules/typeorm/cli",
+    "migration:create": "npm run typeorm migration:create",
+    "migration:run": "npm run typeorm -- -d typeormconfig.ts migration:run -t=false",
+    "migration:revert": "npm run typeorm -- -d typeormconfig.ts migration:revert",
+    "run-migrations-and-start": "npm run migration:run && npm run start:prod"
+  },
+  "dependencies": {
+    "@nestjs/common": "^9.0.0",
```
Comment (paraschivionut) [inline package.json]: Let's keep static versions (w/o ^)

**Review by paraschivionut — CHANGES_REQUESTED** [review]

File: nest-cli.json (hunk 1 above)
Comment (mcristi) [reply nest-cli.json:1]: will rename it to `docs` to be similar to webframe

File: package.json (hunk 2 above)
Comment (mcristi) [reply package.json]: Done

**Review by paraschivionut — APPROVED** [review]

**Review by paraschivionut — APPROVED** [review]

**Pull request merged**